packages/web-components/fast-components/src/design-tokens.ts

```diff
@@ -357,37 +373,40 @@ const accentFillByContrast = (contrast: number) => (
     );
 };
 /** @public */
-export const AccentFill = DI.createInterface<
-    (element: HTMLElement, fill?: SwatchRGB) => ReturnType<typeof accentFillAlgorithm>
->("accent-fill", builder =>
-    builder.instance(accentFillByContrast(ContrastTarget.normal))
-);
+export const accentFillRecipe = create<ColorFamilyRecipe>({
+    name: "accent-fill-recipe",
+    cssCustomPropertyName: null,
+}).withDefault({
+    evaluate: (element: HTMLElement, reference?: Swatch): SwatchFamily =>
+        accentFillByContrast(ContrastTarget.normal)(element),
```

Comment:
I think you need to pass `reference` to the resolved color recipe, otherwise the second argument is pointless

Reply:
Good catch.